product_docs/docs/pem/9/registering_agent.mdx

```diff
@@ -233,3 +233,31 @@ To use a nonroot user account to register a PEM agent, you must first install th
 
 !!! Note
     Any probes and jobs that require root permission or access to a file owned by another user (for example,  enterprisedb) fail.
+
+## Modifying the pemagent service file to update agent configuration file path
```

Comment:
I think this should go in the Troubleshooting section. We can provide the context that if you run into this situation, go to the troubleshooting section and provide a link to that section. Also, I think the heading should be describing what the problem is. This heading sounds more like a solution to the problem.

Reply:
I have added a note in the "Registering a PEM agent" topic indicating the situation/issue the user might face and provided the link to the "Troubleshooting" topic, where the steps have been added. The topic heading has also been changed.